bin/varnishd/storage/stevedore.c

```diff
@@ -96,6 +96,75 @@ STV_NewObject(struct worker *wrk, struct objcore *oc,
 
 /*-------------------------------------------------------------------*/
 
+struct stv_buffer {
+	unsigned		magic;
+#define STV_BUFFER_MAGIC	0xf39cb6c2
+	const struct stevedore	*stv;
+	size_t			size;
+	uintptr_t		priv;
+	uint8_t			buf[];
```

Comment:
The `buf[]` member seems to be unused ?

Reply:
Yeah, it didn't surve much purpose. I've removed it.